### PR TITLE
Adding test cases for txservice

### PR DESCRIPTION
**Pull request by liu-zhipeng**
## Checklist

- [ ] Test manually on `test-ui` using remote chains.
- [ ] Run load tests.
- [ ] Update documentation if needed.
- [ ] Update CHANGELOG.md.

### CODE REVIEW

File: packages/txservice/src/shared/contracts.ts [hunk 1]
```diff
@@ -3,14 +3,22 @@ import contractDeployments from "@connext/nxtp-contracts/deployments.json";
...
+
 /**
  * A number[] list of all chain IDs on which a Connext Price Oracle Contracts
  * have been deployed.
  */
 export const CHAINS_WITH_PRICE_ORACLES: number[] = ((): number[] => {
   const chainIdsForGasFee: number[] = [];
-  Object.keys(contractDeployments).forEach((chainId) => {
-    const record = (contractDeployments as any)[chainId];
+  const _contractDeployments = getContractDeployments();
```
Comment (jakekidd) [inline packages/txservice/src/shared/contracts.ts:19]: what's with the underscore '_'? lol

File: packages/txservice/src/shared/contracts.ts [hunk 2]
```diff
@@ -32,7 +40,8 @@ export const CHAINS_WITH_PRICE_ORACLES: number[] = ((): number[] => {
  * deployed.
  */
 export const getDeployedPriceOracleContract = (chainId: number): { address: string; abi: any } | undefined => {
-  const record = (contractDeployments as any)[String(chainId)] ?? {};
+  const _contractDeployments = getContractDeployments();
+  const record = _contractDeployments[String(chainId)] ?? {};
```
Comment (jakekidd) [inline packages/txservice/src/shared/contracts.ts:44]: I know you didn't write this code, but this should use `chainId.toString()`, not `String(chainId)`

File: packages/txservice/src/shared/contracts.ts (hunk 1 above)
Comment (sanchaymittal) [reply packages/txservice/src/shared/contracts.ts:19]: make sense, we have contractDeployements state in file

File: packages/txservice/src/shared/contracts.ts (hunk 2 above)
Comment (sanchaymittal) [reply packages/txservice/src/shared/contracts.ts:44]: yeah, better practice to use toString(). We were doing this at multiple files, updated all.

**Review by sanchaymittal — APPROVED** [review]

**Pull request merged**